data-access/burgemeester.ts

```diff
@@ -203,27 +198,32 @@ export const benoemBurgemeester = async (
       orgGraph,
       existingMandataris,
       date,
-      burgemeesterMandaat,
+      burgemeesterMandaatUri,
     );
 
-    await endExistingMandataris(orgGraph, existingMandataris, date, benoeming);
+    await endExistingMandataris(
+      orgGraph,
+      existingMandataris,
+      date,
+      benoemingUri,
+    );
   } else {
     // we need to create a new mandataris from scratch
     newMandatarisUri = await createBurgemeesterFromScratch(
       orgGraph,
       burgemeesterUri,
-      burgemeesterMandaat,
+      burgemeesterMandaatUri,
       date,
-      benoeming,
+      benoemingUri,
     );
   }
-  const benoemingUri = sparqlEscapeUri(benoeming);
+  const benoeming = sparqlEscapeUri(benoemingUri);
   await updateSudo(`
```

Comment:
`Breakpoint;`

benoeming is escaped correctly + rename to escapedBenoemingUri when linter allows it